### DISCUSSION TITLE
what is mean <C-+>?

### DISCUSSION THREAD

**Discussion by I-Want-ToBelieve** [tenxsoydev/size-matters.nvim#5] [Q&A] [answered]
https://github.com/tenxsoydev/size-matters.nvim/blob/bc5ae8db3c31eec3b121f03d5de0444ab6a9a983/lua/size-matters/init.lua#L48

ctrl and + ?

**Comment by ttytm** [comment, marked as answer]
Hey, sorry for the delayed response. I didn't get a notification about an issue being opened.

Yes, it's Ctrl and +. Which usually is:
<kbd>Ctrl+Shift+=</kbd>
